tools/prom-metrics-collector/metrics_json_generator.go

```diff
@@ -40,13 +39,8 @@ func main() {
 		panic(err)
 	}
 
-	metricsList := metrics.ListMetrics()
-	for _, metric := range metricsparser.ReadFromPrometheusCR() {
-		metricsList = append(metricsList, metric)
-	}
-
 	var metricFamilies []parser.Metric
-	for _, m := range metricsList {
+	for _, m := range metrics.ListMetrics() {
```

Comment:
where is the recording rules? 
In hco Shirly asked to add them as well to the doc and linter (if the name is sum: or cluster: just ignore them in the list above), see: https://github.com/kubevirt/hostpath-provisioner-operator/pull/417/files#diff-67148ace3c0d839a4031d5be8a050b10553fcb7befa7f5b3a191d9b95ffda424R59

Reply:
updated